### DISCUSSION TITLE
Super excited about 3.0! Is there anywhere that regular info about ETA and progress towards that is available?

### DISCUSSION THREAD

**Discussion by kylebakerio** [requarks/wiki#3038] [General]
I got a 2.x running, but after realizing 3.x is right around the corner, have decided to wait to launch my wiki until then--I really want (must-have) the 'edit in any editor' feature, and don't want to go through an upgrade right at site launch. 

Is there any way to keep tabs / get updates on this? If there's just no record, understood, but if there's a place where we can watch, let me know.

I'm new to wiki.js, but if there's some tasks I could help with anyways, I might be able to as well.

Cheers, and thanks for this great resource. I'm playing with the 2.x install myself and seeding some content I'll copy over to a clean install of 3.x when it comes in the meantime, and really enjoying it.

**Comment by Igorgro** [comment]
You can see some info here: https://blog.js.wiki/

**Reply by tukusejssirs** [reply]
Yes, there are monthly update articles (last from May 2021 though), but no ETA I have found.

@NGPixel, could you please tell us if the (preliminary) v3.0 release date is already known? Thanks in advance and keep up the good work! :+1:

**Reply by tukusejssirs** [reply]
@NGPixel, I’ve found [this comment of yours](https://github.com/Requarks/wiki/pull/2189#issuecomment-689152776), where you say the preliminary release date is Q4 2020, which seems to be missed.

**Reply by benjamincarmel** [reply]
> @NGPixel, I’ve found [this comment of yours](https://github.com/Requarks/wiki/pull/2189#issuecomment-689152776), where you say the preliminary release date is Q4 2020, which seems to be missed.

@NGPixel has asked people not to ask when it's going to be ready there isn't any ETA and he won't provide you with one.

From his May Update:
"Please also refrain from asking about ETAs as the honest answer is simply: I don't know. As explained in the docs, this is mostly a single developer project and it's not my main job."

Simply put he didn't even do a June update and commits have slowed and he's the only developer and there isn't even any really good process for other developers to get involved and most pull requests sit unaddressed. So I'd expect a long wait for even a beta of 3.0 and maybe not even a beta of 3.0 in 2021. Maybe we will get a beta in 2022 if we're lucky.

I think everyone wants 3.0 because 2.x has a lot of bugs and lacks features but we have to be patient when it's single developer projects like this.

**Reply by tukusejssirs** [reply]
Thanks for clarification. I have discovered Wiki.js just recently, so I didn’t know it is a single-man (side-)project.

Still it might be a good idea to have a milestone or something to track the progress towards v3.0 with all issues that @NGPixel wants to fix/address before the release. I personally don’t actually need the ETA date, but rather what needs to be fixed before the release. Currently, I don’t see anything like that.

> I think everyone wants 3.0 because 2.x has a lot of bugs and lacks features but we have to be patient when it's single developer projects like this.

True. I have discovered some of them. But the thing is @NGPixel does not seem to want to backport the fixes to v2.x, which might be a good idea, though I understand that a single dev does not have the time required to do that.
